eng/Versions.props

```diff
@@ -224,7 +224,8 @@
     <IdentityServer4PackageVersion>3.0.0</IdentityServer4PackageVersion>
     <IdentityServer4StoragePackageVersion>3.0.0</IdentityServer4StoragePackageVersion>
     <IdentityServer4EntityFrameworkStoragePackageVersion>3.0.0</IdentityServer4EntityFrameworkStoragePackageVersion>
-    <MessagePackPackageVersion>2.1.80</MessagePackPackageVersion>
+    <MessagePackPackageVersion>2.1.90</MessagePackPackageVersion>
+    <MessagePackAnalyzerPackageVersion>2.1.90</MessagePackAnalyzerPackageVersion>
```

Comment:
You can use just one msbuild property for both versions since we always release them together.

Reply:
is this what you have in mind ?
```
    <MessagePackPackageVersion>2.1.90</MessagePackPackageVersion>
    <MessagePackAnalyzerPackageVersion>$(MessagePackPackageVersion)</MessagePackAnalyzerPackageVersion>
```

Reply:
Sure, you could do that if you wanted to keep two unique msbuild property names. Or you could just reuse the same msbuild property for both package references.

But honestly I don't know why you even have a separate Versions.props file. Why not just write the version directly into your Dependencies.props file?

Reply:
PEBCAK